### DISCUSSION TITLE
Adding tabindex to results?

### DISCUSSION THREAD

**Discussion by bilke** [Pagefind/pagefind#503] [Ideas]
Is it possible to add tabindex to the result elements?

If not this feature request would improve (keyboard) accessibility a lot. I suggest to add `tabindex=0` to all result `<li>`-elements.

**Comment by bglw** [comment]
Not currently possible with the Default UI, though would be possible with the [Modular UI](https://www.npmjs.com/package/@pagefind/modular-ui).

Some sort of keyboard navigation is definitely something I'm keen on officially adding, but I want to make sure it's accessible before doing so — I don't believe `tabindex` on a non-interactive `li` is ideal for screen readers, so it might more closely resemble a combobox with arrow key navigation. Haven't fully specced that out yet 🙂
